### PR TITLE
Change default cost limit 30 $

### PR DESCRIPTION
**Pull request by egenc**
This PR is intended for setting the default cost limit to 30$ to prevent any leakage.

### CODE REVIEW

File: cloudos/__main__.py [hunk 1]
```diff
@@ -128,7 +128,7 @@ def cromwell():
               help='Name of the repository platform of the workflow. Default=github.',
               default='github')
 @click.option('--cost-limit',
-              help='Add a cost limit to your job. Default=-1 (no cost limit).',
+              help='Add a cost limit to your job. Default=30.0 (For no cost limit please use -1).',
               type=float,
               default=-1)
```
Comment (dapineyro) [inline cloudos/__main__.py:133]:
```suggestion
              default=30.0)
```

**Review by dapineyro — CHANGES_REQUESTED** [review]
Hi @egenc , thanks for all the changes. There is still one change required in `click` defaults.

In addition, please, change all the `v1.1.0` references to `v.1.1.1` in `README.md`

**Comment by danielboloc** [comment]
Hi @egenc, looks good. A few suggestions for the PR:

- It would be great to include the cloudOS tests in the PR description. I tested the new Docker image `quay.io/lifebitaiorg/cloudos-cli:v1.1.1` with these jobs:

  - https://cloudos.lifebit.ai/app/jobs/635baadee52b8f015599a01b -> had a `--cost-limit 0.05` and completed succesfully (total cost was $0.0212)
  - https://cloudos.lifebit.ai/app/jobs/635bac5ae52b8f015599a15b (the same parameters as above) -> had a `--cost-limit 0.01` and therefore failed/aborted. It was nice that it send me an email at 80% of reaching the cost limit and one after the limit was reached.

On the cloudOS platform I saw the second job was aborted, but I could not find in the log that the cost limit was exceeded.

So the new image `quay.io/lifebitaiorg/cloudos-cli:v1.1.1` does work properly.

**Review by cgpu — APPROVED** [review]

**Review by cgpu — CHANGES_REQUESTED** [review]
@egenc Change the version and add 1.2.1 it's a small patch

**Review by dapineyro — APPROVED** [review]
All the changes needed for version bumping from `v1.2.0` to `v1.2.1` were made. LGTM!

**Review by cgpu — APPROVED** [review]

**Pull request merged**